cmd/minikube/cmd/start.go

```diff
@@ -1757,8 +1757,17 @@ $ minikube config unset kubernetes-version`)
 	} else if strings.EqualFold(strings.ToLower(paramVersion), "latest") || strings.EqualFold(strings.ToLower(paramVersion), "newest") {
 		paramVersion = constants.NewestKubernetesVersion
 	}
-
-	nvs, err := semver.Make(strings.TrimPrefix(paramVersion, version.VersionPrefix))
+	kubernetesSemver := strings.TrimPrefix(paramVersion, version.VersionPrefix)
```

Comment:
lets make sure we notify the user that we chose latest patch for them since they didn't provide a patch version

Reply:
also a better PR title would be, "--kubernetes-version: assume latest patch ver if not specified by user"

Reply:
Done. I abstracted out the majorMinor check and print it out in validateKubernetesVersion. The function uses the param version so this could be slightly redundant as we do that processing of the param in getKubernetesVersion already and thus redo some of it in validateKubernetesVersion so we can run the check there as well.